docs/docusaurus/docs/core/set_up_a_gx_environment/install_additional_dependencies.md

```diff
@@ -13,7 +13,7 @@ import InstallSql from './_install_additional_dependencies/_sql.md'
 import InstallSpark from './_install_additional_dependencies/_spark.md'
 
 Some environments and Data Sources require additional Python libraries or third party utilities that are not included in the base installation of
-GX Core. Use the information provided here to install the necessary dependencies for Amazon S3, Azure Blob Storage, Google Cloud Storage, and SQL databases.
+GX Core. Use the information provided here to install the necessary dependencies for your databases.
```

Comment:
Note for reviewer: this list was missing one option that's present in the tabs that follow. To reduce maintenance burden for the future I made the intro line more generic so it won't have to be updated any time a new tab is added.